Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14-alpine
+
+ENV WORKDIR=/qboard
+ENV USER=qboard
+ENV GROUP=$USER
+
+WORKDIR $WORKDIR
+
+RUN addgroup -S $GROUP \
+ && adduser -S $USER -G $GROUP -D \
+ && chown $USER:$GROUP $WORKDIR
+USER qboard
+
+COPY ./ .
+ 
+RUN npm install nwb@0.25.2
```

Comment:
Should this not be `npm install`

Reply:
we should probably just lock it at 0.25.2 in package.json? :shrug:

Reply:
Yeah and I think it needs the other dependencies too. (I haven't tried running it yet)

Reply:
I have tried this with the stripped `RUN npm install` and it has worked. I am happy to push in this change in a new PR. I am usually a fan of locking somewhere, wasn't aware you could do this in `package.json`, so that's great. 

I've had many containers break due to the default use of `latest` when no version is specified. Note, the `node:14-alpine` will keep the base image on the 14 branch, whilst still getting the minor updates, but not v15.

Reply:
Have updated this in #20 to be `RUN npm install`.

Reply:
Update: yes, a full install was necessary; not just of nwb. the reason you didn't have any issues with the original version was because you were copying over the entire (existing, local) node_modules directory, so the npm install was a few internal checks/logs and then a no-op.

using npm install is okay as a form of locking. all dependencies correspond to the versions specified in package.json. however, to ignore package.json and use exact version numbers for the entire dependency tree, which are locked in package-lock.json (also commited to git), we use npm ci.

This and some performance issues have been fixed in #90.